tests/resources/Vsphere-Util.robot

```diff
@@ -203,6 +203,7 @@ Add Host To VCenter
     \   ${out}=  Run  govc cluster.add -hostname=${host} -username=${user} -dc=${dc} -password=${pw} -noverify=true
     \   ${status}=  Run Keyword And Return Status  Should Contain  ${out}  OK
     \   Return From Keyword If  ${status}
+    \   Sleep  3 minutes
```

Comment:
Retry seems fine since the resources were just created and, nimbus.  Don't think you need to wait so long in between attempts, but your call.